test/procedures-shared.test.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2022 Cloudcontainers B.V.
+ */
+
+var test = require('tape').test;
+var shared = require('../lib/procedures/shared.js');
+
+test('nextIdNoInstances', function (t) {
+    var arg = {
+        change: {
+            service: {
+                name: "portolan"
```

Comment:
Strings should use single quotes here, and through the rest of this file.

Linting errors:

```
  18:23  error  Strings must use singlequote  quotes
  35:23  error  Strings must use singlequote  quotes
  41:28  error  Strings must use singlequote  quotes
  59:23  error  Strings must use singlequote  quotes
  65:28  error  Strings must use singlequote  quotes
  70:28  error  Strings must use singlequote  quotes
  75:28  error  Strings must use singlequote  quotes
```